### DISCUSSION TITLE
你好，请问是否可以增加按下ctrl（或者alt、shift）键，则忽略特定动作，执行通用动作的功能

### DISCUSSION THREAD

**Discussion by fengshi0519** [shuax/MouseInc.Settings#186] [Ideas]
如题，当某一个设定好的程序中的特定动作和通用动作重复，但是我想在某一时刻执行通用动作的功能的时候，此时按下ctrl（或者alt、shift）键就可以忽略当前特定动作，而执行通用动作呢，谢谢

**Comment by 6Tom** [comment]
那就改特定动作手势吧，兄弟。这事都可以自己换个角度解决的，你提这功能帅的很，我也想要，但是换个思路也能解决，咱们就自己解决问题吧。

比如浏览器的“下右”默认就是“关闭相似窗口”，但是我为了加回“关闭窗口”，就把“下右”设为了“关闭窗口”，然后把“上右”设为了“关闭相似窗口”。
